test/test-duplicate-expressions-code.js

```diff
@@ -0,0 +1,8 @@
+/*1* KEEP THIS CODE AT THE TOP TO AVOID LINE NUMBER CHANGES */
+/*2*/'use strict';
+/*3*/function foo(a) {
+/*4*/  process.nextTick(function() {
+/*5*/    a = 0;
+/*6*/  });
+/*7*/}
+/*8*/module.exports = foo;
```

Comment:
To prevent merge conflicts with landing the upcoming PRs, I won't fix this in this PR.  I'll record this so we don't forget to add EOL for all the `-code.js` files after the PRs have all landed.